src/Catty/Setup/CatrobatSetup+Bricks.swift

```diff
@@ -137,62 +137,85 @@
                           name: kLocalizedCategoryEvent,
                           color: UIColor.eventBrick,
-                          strokeColor: UIColor.eventBrickStroke),
+                          strokeColor: UIColor.eventBrickStroke,
+                          enabled: true),
 
             BrickCategory(type: kBrickCategoryType.controlBrick,
                           name: kLocalizedCategoryControl,
                           color: UIColor.controlBrickOrange,
-                          strokeColor: UIColor.controlBrickStroke),
+                          strokeColor: UIColor.controlBrickStroke,
+                          enabled: true),
 
             BrickCategory(type: kBrickCategoryType.motionBrick,
                           name: kLocalizedCategoryMotion,
                           color: UIColor.motionBrickBlue,
-                          strokeColor: UIColor.motionBrickStroke),
+                          strokeColor: UIColor.motionBrickStroke,
+                          enabled: true),
 
             BrickCategory(type: kBrickCategoryType.lookBrick,
                           name: kLocalizedCategoryLook,
                           color: UIColor.lookBrickGreen,
-                          strokeColor: UIColor.lookBrickStroke),
+                          strokeColor: UIColor.lookBrickStroke,
+                          enabled: true),
 
             BrickCategory(type: kBrickCategoryType.penBrick,
                           name: kLocalizedCategoryPen,
                           color: UIColor.penBrickGreen,
-                          strokeColor: UIColor.penBrickStroke),
+                          strokeColor: UIColor.penBrickStroke,
+                          enabled: true),
 
             BrickCategory(type: kBrickCategoryType.soundBrick,
                           name: kLocalizedCategorySound,
                           color: UIColor.soundBrickViolet,
-                          strokeColor: UIColor.soundBrickStroke),
+                          strokeColor: UIColor.soundBrickStroke,
+                          enabled: true),
 
             BrickCategory(type: kBrickCategoryType.dataBrick,
                           name: kLocalizedCategoryData,
                           color: UIColor.variableBrickRed,
-                          strokeColor: UIColor.variableBrickStroke)
+                          strokeColor: UIColor.variableBrickStroke,
+                          enabled: true)
         ]
 
         if isPhiroEnabled() {
             categories.prepend(BrickCategory(type: kBrickCategoryType.phiroBrick,
                                              name: kLocalizedCategoryPhiro,
                                              color: UIColor.phiroBrick,
-                                             strokeColor: UIColor.phiroBrickStroke))
+                                             strokeColor: UIColor.phiroBrickStroke,
+                                             enabled: true))
         }
         if isArduinoEnabled() {
```

Comment:
Please remove those code duplication. Suggestion: add boolean directly to argument.